packages/libs/client/src/interfaces/.gitignore

```diff
@@ -0,0 +1 @@
+async-pipe-type.ts
```

Comment:
Should this be ignored? It's part of the public API now, right?

Reply:
it is generated by the `create-async-pipe-type.ts` during build process, so it will be in the public API.